src/js/common/components/FAQBody.jsx

```diff
@@ -4,6 +4,7 @@ import styled from 'styled-components';
 import { isCordova, isWebApp } from '../utils/isCordovaOrWebApp';
 import { renderLog } from '../utils/logging';
 import ToolBar from './Widgets/ToolBar';
+import webAppConfig from "../../config";
```

Comment:
Thank you @bonniehuynh for this pull request! Make sure to run `npm run lint` before making pull requests so you can catch ESLINT errors like this (these double-quotes should be single quotes). 👍